ASP.NET Core_EF_CodeFirst/Controllers/AuthorizedController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BLL.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ASP.NET_Core_EF_CodeFirst.Controllers
+{
+    //[Route("api/[controller]")]
+    //[ApiController]
+    //public class AuthorizedController : ControllerBase
```

Comment:
Please uncomment the code and just add [Authorize] attribute to the class and inherit this class for all the controllers except for AccountController, which doesn't need It.